src/components/Shared/MessageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import getAvatar from '@lib/getAvatar';
+import type { FC } from 'react';
+import { useState } from 'react';
+import { useAppStore } from 'src/store/app';
+
+import Follow from './Follow';
+
+const MessageHeader: FC = () => {
+  const [following, setFollowing] = useState(false);
+  const currentProfile = useAppStore((state) => state.currentProfile);
```

Comment:
Shouldn't this be the other person's profile, not your own?

Reply:
Yes, was waiting for Elise's PR to get merged, this is all sorted now